source/CoordinateConversion/CoordinateConversionLibrary/Behaviors/WindowBehavior.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
```

Comment:
Maybe also a short comment to explain why this solution is necessary - i.e. WPF doesn't expose this window style